packages/core-bridge/Cargo.toml

```diff
@@ -17,16 +17,19 @@ lto = true
 incremental = false
 
 [dependencies]
+async-trait = "0.1.83"
 futures = { version = "0.3", features = ["executor"] }
 log = "0.4"
-neon = { version = "1.0.0", default-features = false, features = ["napi-6"] }
+neon = { version = "1.0.0", default-features = false, features = ["napi-6", "futures"] }
 opentelemetry = "0.24"
 parking_lot = "0.12"
 prost = "0.13"
 prost-types = "0.13"
 serde_json = "1.0"
 tokio = "1.13"
 once_cell = "1.19"
-temporal-sdk-core = { version = "*", path = "./sdk-core/core", features = ["ephemeral-server"] }
-temporal-client = { version = "*", path = "./sdk-core/client" }
+temporal-sdk-core = { version = "*", path = "../../../sdk-core/core", features = ["ephemeral-server"] }
+temporal-client = { version = "*", path = "../../../sdk-core/client" }
+#temporal-sdk-core = { version = "*", path = "./sdk-core/core", features = ["ephemeral-server"] }
```

Comment:
Leftover? lines 33 and 34

Reply:
I've left it here on purpose until https://github.com/temporalio/sdk-core/pull/838 is merged